app/assets/stylesheets/base/_forms.scss

```diff
@@ -72,7 +72,12 @@ input[type="file"] {
 }
 
 select {
+  background-color: white;
+  border: 1px solid #ddd;
+  border-radius: 3px;
+  box-shadow: inset 0 1px 3px rgba(0, 0, 0, 0.06);
+  box-sizing: border-box;
+  width: 100%;
```

Comment:
Properties should be ordered background-color, border, border-radius, box-shadow, box-sizing, height, margin-bottom, width